cli/util/send.go

```diff
@@ -37,6 +40,17 @@ func sendTx(ctx *cli.Context) error {
 	if err != nil {
 		return cli.NewExitError(fmt.Errorf("failed to submit transaction to RPC node: %w", err), 1)
 	}
-	fmt.Fprintln(ctx.App.Writer, res.StringLE())
+	var aer *state.AppExecResult
+	if ctx.Bool("await") {
+		version, err := c.GetVersion()
+		if err != nil {
+			return cli.NewExitError(fmt.Errorf("failed to get version from RPC node: %w", err), 1)
+		}
+		aer, err = waiter.NewWaiter(c, version).Wait(tx.Hash(), tx.ValidUntilBlock, nil)
```

Comment:
Use `err` from the previous call as the last argument to `Wait`, then you don't need to handle it manualy.